### DISCUSSION TITLE
Frustrated with Student Pack Application Issues

### DISCUSSION THREAD

**Discussion by K-Khushal** [community/community#106685] [GitHub Education]
### Hi everyone,

I'm writing to express my frustration with the mobile while attempting to apply for the GitHub Student Developer Pack. While eager to access the benefits, the process was hindered by several technical glitches that made it nearly impossible to complete.

- Excessive Steps & Redundancy:
The application required unnecessary steps like adding 2FA and billing information, even though it's a free student pack. This extra work felt confusing and delayed my progress.

- Mobile App Camera Woes:
I couldn't select the camera using the mobile app, making capturing the required verification photo impossible. Additionally, the website froze randomly, further adding to my frustration.

- Limited Verification Options:
My student ID alone wasn't enough, but it offered no way to upload supporting documents. This lack of flexibility prevented me from completing the application despite meeting the eligibility criteria.

These issues significantly impacted my experience and prevented me from securing the valuable resources offered by the Student Pack. I kindly request your urgent attention to the following:

Streamlining the application process, especially for student applicants.
Ensuring full functionality of the mobile camera selection and fixing any freezing issues.
Implementing an option to upload additional verification documents.

 I value your time and appreciate your quick assistance in resolving these matters.

Sincerely,
Khushal Khandelwal

**Comment by burekichevapi** [comment]
I had such a frustrating process that I just gave up. I go to school online and don't have an ID. So, instead of having an option to UPLOAD A DOCUMENT, they only give the option to take a damn picture, why?! Then, after having to print my transcript, they wanted me to enable MFA, then I had to take a detour to do that. Mind you, all of this required me to take the picture of the transcript over and over again after every failed attempt. Then, they wanted me to fill out billing info, again why!? Its supposed to be free!!! After 4 failed attempt and an hour of my life wasted, I just gave up. I used to have this Education Pack a few years back and it was NOT this convoluted.

If MS is trying to discourage students from using their services, then its a job well done!

**Comment by r21ritika** [comment]
I also had the same experience. I am pursuing an online master's programme and it did not let me upload transcripts or admission letter for verification. Instead, I had to try multiple times with mobile front camera to click a clearer picture of transcript on my laptop screen. They should really change this stupid process for student verification.
